app/src/main/cpp/skyline/jvm.cpp

```diff
@@ -104,7 +109,41 @@ namespace skyline {
         env->CallVoidMethod(instance, clearVibrationDeviceId, index);
     }
 
+    jobject JvmManager::ShowKeyboard(applet::swkbd::KeyboardConfigVB &config, std::u16string initial_text) {
+        auto buff{env->NewDirectByteBuffer(&config, sizeof(applet::swkbd::KeyboardConfigVB))};
```

Comment:
The shortened version of `buffer` would be `buf` not `buff` unless the `buffer` is a `buff buffer` in that case `buff` may be appropriate but I am unsure if a JNI `DirectByteBuffer` may be classified as such. I do think that given no name overlap, this should just be named `buffer` rather than being pointlessly shortened.